clients/src/main/java/org/apache/kafka/common/requests/ApiVersionsRequest.java

```diff
@@ -31,18 +31,44 @@
 public class ApiVersionsRequest extends AbstractRequest {
 
     public static class Builder extends AbstractRequest.Builder<ApiVersionsRequest> {
-        private static final String DEFAULT_CLIENT_SOFTWARE_NAME = "apache-kafka-java";
+        private static final String DEFAULT_CLIENT_SOFTWARE_NAME = "linkedin-kafka-java";
 
-        private static final ApiVersionsRequestData DATA = new ApiVersionsRequestData()
-            .setClientSoftwareName(DEFAULT_CLIENT_SOFTWARE_NAME)
-            .setClientSoftwareVersion(AppInfoParser.getVersion());
+        private final ApiVersionsRequestData DATA;
```

Comment:
but now that its not static you cant name it in all-caps :-). also, "data" is a little vague :-)

Reply:
fixed